### DISCUSSION TITLE
Welcome to PCAPdroid Discussions!

### DISCUSSION THREAD

**Discussion by emanuele-f** [emanuele-f/PCAPdroid#389] [Announcements]
You can now use Discussions as an alternative to creating issues when you want to share an idea, ask something, and provide your feedback. If you prefer a live discussion, you can join the PCAPdroid community [on Telegram](https://t.me/PCAPdroid) or [on Matrix](https://matrix.to/#/#pcapdroid:matrix.org)

**Comment by katastrophe92** [comment]
Hey there! Can it be used as a adblocker also? Thanks

**Reply by emanuele-f** [reply]
You can use the firewall feature to block ads, but you have to manually create the rules

**Reply by anpic** [reply]
> create the rules

And only Global rules ;)

**Comment by katastrophe92** [comment]
ok thanks

**Comment by berniland** [comment]
Hi Emanuele,
could you please confirm that pcap could sniffe app used bluetooth and wifi in same time ?  
that's a good think that you live in italy because I have to decode maestro+ "protocole" used by your
MCZ stuff ;). unfortunalty MCZ don't provide api or any information to help to make  domotic link.
good day

**Reply by emanuele-f** [reply]
Hello, PCAPdroid can only sniff IP data, not bluetooth

**Comment by berniland** [comment]
Hi, right. it seem I can complete with native Bluetooth HCI snoop log that wireshark could analyse.
I will install pcapdroid and mitm plugin to test

**Comment by totestphonespr-creator** [comment]
Hi Dev and everyone... I have several questions. 

There is a limited number of licenses we can generate after paying for the unlock token. I believe the limit is 3, and I have already got 3 devices using the premium features. This is my question.... If I buy a new phone to replace one of my 3 devices, how can I get another license for that device and future ones? Do I need to pay again for another unlock token or can I request a reset to the limit by providing proof of purchase? Is there a place where we can do that reset if required?

**Reply by emanuele-f** [reply]
Hello, please email me in these cases, this needs to be handled manually because licenses cannot be revoked once generated

**Reply by totestphonespr-creator** [reply]
Thanks, I will send an email with the proof of purchase. Somehow one of my devices that was already setup got back to non-premium and I believe it got to the Google version, not sure if Google Play did an update and caused the issue. Now I will need to reinstall and probably I will not be able to use the licenses I saved. It would be nice to have a way to correct the issue without taking your time, but I appreciate your response.

**Reply by emanuele-f** [reply]
I assume you had the GitHub version installed. The app will be automatically updated by Google Play to the latest version on Google Play. To prevent this, disable the auto-update for the app on Google Play from the three dots menu.

To restore the paid features, just install the latest version from GitHub **without uninstalling** the existing app (otherwise that would erase the features)
